src/ziggurat/kafka_consumer/consumer_handler.clj

```diff
@@ -49,16 +50,19 @@
         batch-size          (count batch)]
     (try
       (when (not-empty batch)
-        (log/infof "[Consumer Group: %s] Processing the batch with %d messages" topic-entity batch-size)
-        (let [start-time             (Instant/now)
-              result                 (batch-handler batch)
-              time-taken-in-millis   (.toMillis (Duration/between start-time (Instant/now)))]
+        (clog/info {:batch-size batch-size} (format "[Consumer Group: %s] Processing the batch with %d messages" topic-entity batch-size))
+        ;(log/infof "[Consumer Group: %s] Processing the batch with %d messages" topic-entity batch-size)
+        (let [start-time           (Instant/now)
+              result               (batch-handler batch)
+              time-taken-in-millis (.toMillis (Duration/between start-time (Instant/now)))]
           (validate-batch-processing-result result)
           (let [messages-to-be-retried (:retry result)
                 to-be-retried-count    (count messages-to-be-retried)
                 skip-count             (count (:skip result))
                 success-count          (- batch-size (+ to-be-retried-count skip-count))]
-            (log/infof "[Consumer Group: %s] Processed the batch with success: [%d], skip: [%d] and retries: [%d] \n" topic-entity success-count skip-count to-be-retried-count)
+
+            (clog/info {:messages-successfully-processed success-count :messages-skipped skip-count :messages-to-be-retried to-be-retried-count} (format "[Consumer Group: %s] Processed the batch with success: [%d], skip: [%d] and retries: [%d] \n" topic-entity success-count skip-count to-be-retried-count))
+            ;(log/infof "[Consumer Group: %s] Processed the batch with success: [%d], skip: [%d] and retries: [%d] \n" topic-entity success-count skip-count to-be-retried-count)
```

Comment:
Please remove the comment. Have we kept it here intentionally?

Reply:
Removed.